apps/emqx/i18n/emqx_schema_i18n.conf

```
@@ -17,8 +17,8 @@ emqx_schema {
       zh: "消息队列的最大长度"
     }
     label {
-      en: "Maximum message queue length."
-      zh: "消息队列的最大长度"
+      en: "Maximum mailbox queue length of process."
```

```suggestion
      en: "Maximum number of messages in the mailbox of the process."
```
I think this sounds a bit more natural, but it's your call :)

---

Thanks for your improvement